.github/workflows/build-and-push-docker.yml

```diff
@@ -21,13 +21,14 @@ jobs:
       - name: Login to DockerHub
         uses: docker/login-action@v1 
         with:
+          registry: pgc-images.sbgenomics.com
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
       - name: Build and push
         id: docker_build
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: kfdrc/open-pedcan:latest
+          tags: pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:latest
```

Comment:
I don't think `pgc-images.sbgenomics.com/` will work here. Tags should be `<username>/<appname>:<version>`